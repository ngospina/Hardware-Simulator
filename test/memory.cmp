| time |       inh        |       inl        |    a    | w |       outh       |       outl       |
| 1    | 0000000000000000 | 0000000000000000 | 0000000 | 1 | 0000000000000000 | 0000000000000000 |
| 2    | 0001000000100000 | 0100000010000001 | 0000001 | 1 | 0001000000100000 | 0100000010000001 |
| 3    | 0010000001000000 | 1000000100000010 | 0000010 | 1 | 0010000001000000 | 1000000100000010 |
| 4    | 0011000001100000 | 1100000110000011 | 0000011 | 1 | 0011000001100000 | 1100000110000011 |
| 5    | 0100000010000001 | 0000001000000100 | 0000100 | 1 | 0100000010000001 | 0000001000000100 |
| 6    | 0101000010100001 | 0100001010000101 | 0000101 | 1 | 0101000010100001 | 0100001010000101 |
| 7    | 0110000011000001 | 1000001100000110 | 0000110 | 1 | 0110000011000001 | 1000001100000110 |
| 8    | 0111000011100001 | 1100001110000111 | 0000111 | 1 | 0111000011100001 | 1100001110000111 |
| 9    | 1000000100000010 | 0000010000001000 | 0001000 | 1 | 1000000100000010 | 0000010000001000 |
| 10   | 1001000100100010 | 0100010010001001 | 0001001 | 1 | 1001000100100010 | 0100010010001001 |
| 11   | 1010000101000010 | 1000010100001010 | 0001010 | 1 | 1010000101000010 | 1000010100001010 |
| 12   | 1011000101100010 | 1100010110001011 | 0001011 | 1 | 1011000101100010 | 1100010110001011 |
| 13   | 1100000110000011 | 0000011000001100 | 0001100 | 1 | 1100000110000011 | 0000011000001100 |
| 14   | 1101000110100011 | 0100011010001101 | 0001101 | 1 | 1101000110100011 | 0100011010001101 |
| 15   | 1110000111000011 | 1000011100001110 | 0001110 | 1 | 1110000111000011 | 1000011100001110 |
| 16   | 1111000111100011 | 1100011110001111 | 0001111 | 1 | 1111000111100011 | 1100011110001111 |
| 17   | 0000001000000100 | 0000100000010000 | 0010000 | 1 | 0000001000000100 | 0000100000010000 |
| 18   | 0001001000100100 | 0100100010010001 | 0010001 | 1 | 0001001000100100 | 0100100010010001 |
| 19   | 0010001001000100 | 1000100100010010 | 0010010 | 1 | 0010001001000100 | 1000100100010010 |
| 20   | 0011001001100100 | 1100100110010011 | 0010011 | 1 | 0011001001100100 | 1100100110010011 |
| 21   | 0100001010000101 | 0000101000010100 | 0010100 | 1 | 0100001010000101 | 0000101000010100 |
| 22   | 0101001010100101 | 0100101010010101 | 0010101 | 1 | 0101001010100101 | 0100101010010101 |
| 23   | 0110001011000101 | 1000101100010110 | 0010110 | 1 | 0110001011000101 | 1000101100010110 |
| 24   | 0111001011100101 | 1100101110010111 | 0010111 | 1 | 0111001011100101 | 1100101110010111 |
| 25   | 1000001100000110 | 0000110000011000 | 0011000 | 1 | 1000001100000110 | 0000110000011000 |
| 26   | 1001001100100110 | 0100110010011001 | 0011001 | 1 | 1001001100100110 | 0100110010011001 |
| 27   | 1010001101000110 | 1000110100011010 | 0011010 | 1 | 1010001101000110 | 1000110100011010 |
| 28   | 1011001101100110 | 1100110110011011 | 0011011 | 1 | 1011001101100110 | 1100110110011011 |
| 29   | 1100001110000111 | 0000111000011100 | 0011100 | 1 | 1100001110000111 | 0000111000011100 |
| 30   | 1101001110100111 | 0100111010011101 | 0011101 | 1 | 1101001110100111 | 0100111010011101 |
| 31   | 1110001111000111 | 1000111100011110 | 0011110 | 1 | 1110001111000111 | 1000111100011110 |
| 32   | 1111001111100111 | 1100111110011111 | 0011111 | 1 | 1111001111100111 | 1100111110011111 |
| 33   | 0000010000001000 | 0001000000100000 | 0100000 | 1 | 0000010000001000 | 0001000000100000 |
| 34   | 0001010000101000 | 0101000010100001 | 0100001 | 1 | 0001010000101000 | 0101000010100001 |
| 35   | 0010010001001000 | 1001000100100010 | 0100010 | 1 | 0010010001001000 | 1001000100100010 |
| 36   | 0011010001101000 | 1101000110100011 | 0100011 | 1 | 0011010001101000 | 1101000110100011 |
| 37   | 0100010010001001 | 0001001000100100 | 0100100 | 1 | 0100010010001001 | 0001001000100100 |
| 38   | 0101010010101001 | 0101001010100101 | 0100101 | 1 | 0101010010101001 | 0101001010100101 |
| 39   | 0110010011001001 | 1001001100100110 | 0100110 | 1 | 0110010011001001 | 1001001100100110 |
| 40   | 0111010011101001 | 1101001110100111 | 0100111 | 1 | 0111010011101001 | 1101001110100111 |
| 41   | 1000010100001010 | 0001010000101000 | 0101000 | 1 | 1000010100001010 | 0001010000101000 |
| 42   | 1001010100101010 | 0101010010101001 | 0101001 | 1 | 1001010100101010 | 0101010010101001 |
| 43   | 1010010101001010 | 1001010100101010 | 0101010 | 1 | 1010010101001010 | 1001010100101010 |
| 44   | 1011010101101010 | 1101010110101011 | 0101011 | 1 | 1011010101101010 | 1101010110101011 |
| 45   | 1100010110001011 | 0001011000101100 | 0101100 | 1 | 1100010110001011 | 0001011000101100 |
| 46   | 1101010110101011 | 0101011010101101 | 0101101 | 1 | 1101010110101011 | 0101011010101101 |
| 47   | 1110010111001011 | 1001011100101110 | 0101110 | 1 | 1110010111001011 | 1001011100101110 |
| 48   | 1111010111101011 | 1101011110101111 | 0101111 | 1 | 1111010111101011 | 1101011110101111 |
| 49   | 0000011000001100 | 0001100000110000 | 0110000 | 1 | 0000011000001100 | 0001100000110000 |
| 50   | 0001011000101100 | 0101100010110001 | 0110001 | 1 | 0001011000101100 | 0101100010110001 |
| 51   | 0010011001001100 | 1001100100110010 | 0110010 | 1 | 0010011001001100 | 1001100100110010 |
| 52   | 0011011001101100 | 1101100110110011 | 0110011 | 1 | 0011011001101100 | 1101100110110011 |
| 53   | 0100011010001101 | 0001101000110100 | 0110100 | 1 | 0100011010001101 | 0001101000110100 |
| 54   | 0101011010101101 | 0101101010110101 | 0110101 | 1 | 0101011010101101 | 0101101010110101 |
| 55   | 0110011011001101 | 1001101100110110 | 0110110 | 1 | 0110011011001101 | 1001101100110110 |
| 56   | 0111011011101101 | 1101101110110111 | 0110111 | 1 | 0111011011101101 | 1101101110110111 |
| 57   | 1000011100001110 | 0001110000111000 | 0111000 | 1 | 1000011100001110 | 0001110000111000 |
| 58   | 1001011100101110 | 0101110010111001 | 0111001 | 1 | 1001011100101110 | 0101110010111001 |
| 59   | 1010011101001110 | 1001110100111010 | 0111010 | 1 | 1010011101001110 | 1001110100111010 |
| 60   | 1011011101101110 | 1101110110111011 | 0111011 | 1 | 1011011101101110 | 1101110110111011 |
| 61   | 1100011110001111 | 0001111000111100 | 0111100 | 1 | 1100011110001111 | 0001111000111100 |
| 62   | 1101011110101111 | 0101111010111101 | 0111101 | 1 | 1101011110101111 | 0101111010111101 |
| 63   | 1110011111001111 | 1001111100111110 | 0111110 | 1 | 1110011111001111 | 1001111100111110 |
| 64   | 1111011111101111 | 1101111110111111 | 0111111 | 1 | 1111011111101111 | 1101111110111111 |
| 65   | 0000100000010000 | 0010000001000000 | 1000000 | 1 | 0000100000010000 | 0010000001000000 |
| 66   | 0001100000110000 | 0110000011000001 | 1000001 | 1 | 0001100000110000 | 0110000011000001 |
| 67   | 0010100001010000 | 1010000101000010 | 1000010 | 1 | 0010100001010000 | 1010000101000010 |
| 68   | 0011100001110000 | 1110000111000011 | 1000011 | 1 | 0011100001110000 | 1110000111000011 |
| 69   | 0100100010010001 | 0010001001000100 | 1000100 | 1 | 0100100010010001 | 0010001001000100 |
| 70   | 0101100010110001 | 0110001011000101 | 1000101 | 1 | 0101100010110001 | 0110001011000101 |
| 71   | 0110100011010001 | 1010001101000110 | 1000110 | 1 | 0110100011010001 | 1010001101000110 |
| 72   | 0111100011110001 | 1110001111000111 | 1000111 | 1 | 0111100011110001 | 1110001111000111 |
| 73   | 1000100100010010 | 0010010001001000 | 1001000 | 1 | 1000100100010010 | 0010010001001000 |
| 74   | 1001100100110010 | 0110010011001001 | 1001001 | 1 | 1001100100110010 | 0110010011001001 |
| 75   | 1010100101010010 | 1010010101001010 | 1001010 | 1 | 1010100101010010 | 1010010101001010 |
| 76   | 1011100101110010 | 1110010111001011 | 1001011 | 1 | 1011100101110010 | 1110010111001011 |
| 77   | 1100100110010011 | 0010011001001100 | 1001100 | 1 | 1100100110010011 | 0010011001001100 |
| 78   | 1101100110110011 | 0110011011001101 | 1001101 | 1 | 1101100110110011 | 0110011011001101 |
| 79   | 1110100111010011 | 1010011101001110 | 1001110 | 1 | 1110100111010011 | 1010011101001110 |
| 80   | 1111100111110011 | 1110011111001111 | 1001111 | 1 | 1111100111110011 | 1110011111001111 |
| 81   | 0000101000010100 | 0010100001010000 | 1010000 | 1 | 0000101000010100 | 0010100001010000 |
| 82   | 0001101000110100 | 0110100011010001 | 1010001 | 1 | 0001101000110100 | 0110100011010001 |
| 83   | 0010101001010100 | 1010100101010010 | 1010010 | 1 | 0010101001010100 | 1010100101010010 |
| 84   | 0011101001110100 | 1110100111010011 | 1010011 | 1 | 0011101001110100 | 1110100111010011 |
| 85   | 0100101010010101 | 0010101001010100 | 1010100 | 1 | 0100101010010101 | 0010101001010100 |
| 86   | 0101101010110101 | 0110101011010101 | 1010101 | 1 | 0101101010110101 | 0110101011010101 |
| 87   | 0110101011010101 | 1010101101010110 | 1010110 | 1 | 0110101011010101 | 1010101101010110 |
| 88   | 0111101011110101 | 1110101111010111 | 1010111 | 1 | 0111101011110101 | 1110101111010111 |
| 89   | 1000101100010110 | 0010110001011000 | 1011000 | 1 | 1000101100010110 | 0010110001011000 |
| 90   | 1001101100110110 | 0110110011011001 | 1011001 | 1 | 1001101100110110 | 0110110011011001 |
| 91   | 1010101101010110 | 1010110101011010 | 1011010 | 1 | 1010101101010110 | 1010110101011010 |
| 92   | 1011101101110110 | 1110110111011011 | 1011011 | 1 | 1011101101110110 | 1110110111011011 |
| 93   | 1100101110010111 | 0010111001011100 | 1011100 | 1 | 1100101110010111 | 0010111001011100 |
| 94   | 1101101110110111 | 0110111011011101 | 1011101 | 1 | 1101101110110111 | 0110111011011101 |
| 95   | 1110101111010111 | 1010111101011110 | 1011110 | 1 | 1110101111010111 | 1010111101011110 |
| 96   | 1111101111110111 | 1110111111011111 | 1011111 | 1 | 1111101111110111 | 1110111111011111 |
| 97   | 0000110000011000 | 0011000001100000 | 1100000 | 1 | 0000110000011000 | 0011000001100000 |
| 98   | 0001110000111000 | 0111000011100001 | 1100001 | 1 | 0001110000111000 | 0111000011100001 |
| 99   | 0010110001011000 | 1011000101100010 | 1100010 | 1 | 0010110001011000 | 1011000101100010 |
| 100  | 0011110001111000 | 1111000111100011 | 1100011 | 1 | 0011110001111000 | 1111000111100011 |
| 101  | 0100110010011001 | 0011001001100100 | 1100100 | 1 | 0100110010011001 | 0011001001100100 |
| 102  | 0101110010111001 | 0111001011100101 | 1100101 | 1 | 0101110010111001 | 0111001011100101 |
| 103  | 0110110011011001 | 1011001101100110 | 1100110 | 1 | 0110110011011001 | 1011001101100110 |
| 104  | 0111110011111001 | 1111001111100111 | 1100111 | 1 | 0111110011111001 | 1111001111100111 |
| 105  | 1000110100011010 | 0011010001101000 | 1101000 | 1 | 1000110100011010 | 0011010001101000 |
| 106  | 1001110100111010 | 0111010011101001 | 1101001 | 1 | 1001110100111010 | 0111010011101001 |
| 107  | 1010110101011010 | 1011010101101010 | 1101010 | 1 | 1010110101011010 | 1011010101101010 |
| 108  | 1011110101111010 | 1111010111101011 | 1101011 | 1 | 1011110101111010 | 1111010111101011 |
| 109  | 1100110110011011 | 0011011001101100 | 1101100 | 1 | 1100110110011011 | 0011011001101100 |
| 110  | 1101110110111011 | 0111011011101101 | 1101101 | 1 | 1101110110111011 | 0111011011101101 |
| 111  | 1110110111011011 | 1011011101101110 | 1101110 | 1 | 1110110111011011 | 1011011101101110 |
| 112  | 1111110111111011 | 1111011111101111 | 1101111 | 1 | 1111110111111011 | 1111011111101111 |
| 113  | 0000111000011100 | 0011100001110000 | 1110000 | 1 | 0000111000011100 | 0011100001110000 |
| 114  | 0001111000111100 | 0111100011110001 | 1110001 | 1 | 0001111000111100 | 0111100011110001 |
| 115  | 0010111001011100 | 1011100101110010 | 1110010 | 1 | 0010111001011100 | 1011100101110010 |
| 116  | 0011111001111100 | 1111100111110011 | 1110011 | 1 | 0011111001111100 | 1111100111110011 |
| 117  | 0100111010011101 | 0011101001110100 | 1110100 | 1 | 0100111010011101 | 0011101001110100 |
| 118  | 0101111010111101 | 0111101011110101 | 1110101 | 1 | 0101111010111101 | 0111101011110101 |
| 119  | 0110111011011101 | 1011101101110110 | 1110110 | 1 | 0110111011011101 | 1011101101110110 |
| 120  | 0111111011111101 | 1111101111110111 | 1110111 | 1 | 0111111011111101 | 1111101111110111 |
| 121  | 1000111100011110 | 0011110001111000 | 1111000 | 1 | 1000111100011110 | 0011110001111000 |
| 122  | 1001111100111110 | 0111110011111001 | 1111001 | 1 | 1001111100111110 | 0111110011111001 |
| 123  | 1010111101011110 | 1011110101111010 | 1111010 | 1 | 1010111101011110 | 1011110101111010 |
| 124  | 1011111101111110 | 1111110111111011 | 1111011 | 1 | 1011111101111110 | 1111110111111011 |
| 125  | 1100111110011111 | 0011111001111100 | 1111100 | 1 | 1100111110011111 | 0011111001111100 |
| 126  | 1101111110111111 | 0111111011111101 | 1111101 | 1 | 1101111110111111 | 0111111011111101 |
| 127  | 1110111111011111 | 1011111101111110 | 1111110 | 1 | 1110111111011111 | 1011111101111110 |
| 128  | 1111111111111111 | 1111111111111111 | 1111111 | 1 | 1111111111111111 | 1111111111111111 |
| 129  | 1111111111111111 | 1111111111111111 | 0000000 | 0 | 0000000000000000 | 0000000000000000 |
| 130  | 1111111111111111 | 1111111111111111 | 0000001 | 0 | 0001000000100000 | 0100000010000001 |
| 131  | 1111111111111111 | 1111111111111111 | 0000010 | 0 | 0010000001000000 | 1000000100000010 |
| 132  | 1111111111111111 | 1111111111111111 | 0000011 | 0 | 0011000001100000 | 1100000110000011 |
| 133  | 1111111111111111 | 1111111111111111 | 0000100 | 0 | 0100000010000001 | 0000001000000100 |
| 134  | 1111111111111111 | 1111111111111111 | 0000101 | 0 | 0101000010100001 | 0100001010000101 |
| 135  | 1111111111111111 | 1111111111111111 | 0000110 | 0 | 0110000011000001 | 1000001100000110 |
| 136  | 1111111111111111 | 1111111111111111 | 0000111 | 0 | 0111000011100001 | 1100001110000111 |
| 137  | 1111111111111111 | 1111111111111111 | 0001000 | 0 | 1000000100000010 | 0000010000001000 |
| 138  | 1111111111111111 | 1111111111111111 | 0001001 | 0 | 1001000100100010 | 0100010010001001 |
| 139  | 1111111111111111 | 1111111111111111 | 0001010 | 0 | 1010000101000010 | 1000010100001010 |
| 140  | 1111111111111111 | 1111111111111111 | 0001011 | 0 | 1011000101100010 | 1100010110001011 |
| 141  | 1111111111111111 | 1111111111111111 | 0001100 | 0 | 1100000110000011 | 0000011000001100 |
| 142  | 1111111111111111 | 1111111111111111 | 0001101 | 0 | 1101000110100011 | 0100011010001101 |
| 143  | 1111111111111111 | 1111111111111111 | 0001110 | 0 | 1110000111000011 | 1000011100001110 |
| 144  | 1111111111111111 | 1111111111111111 | 0001111 | 0 | 1111000111100011 | 1100011110001111 |
| 145  | 1111111111111111 | 1111111111111111 | 0010000 | 0 | 0000001000000100 | 0000100000010000 |
| 146  | 1111111111111111 | 1111111111111111 | 0010001 | 0 | 0001001000100100 | 0100100010010001 |
| 147  | 1111111111111111 | 1111111111111111 | 0010010 | 0 | 0010001001000100 | 1000100100010010 |
| 148  | 1111111111111111 | 1111111111111111 | 0010011 | 0 | 0011001001100100 | 1100100110010011 |
| 149  | 1111111111111111 | 1111111111111111 | 0010100 | 0 | 0100001010000101 | 0000101000010100 |
| 150  | 1111111111111111 | 1111111111111111 | 0010101 | 0 | 0101001010100101 | 0100101010010101 |
| 151  | 1111111111111111 | 1111111111111111 | 0010110 | 0 | 0110001011000101 | 1000101100010110 |
| 152  | 1111111111111111 | 1111111111111111 | 0010111 | 0 | 0111001011100101 | 1100101110010111 |
| 153  | 1111111111111111 | 1111111111111111 | 0011000 | 0 | 1000001100000110 | 0000110000011000 |
| 154  | 1111111111111111 | 1111111111111111 | 0011001 | 0 | 1001001100100110 | 0100110010011001 |
| 155  | 1111111111111111 | 1111111111111111 | 0011010 | 0 | 1010001101000110 | 1000110100011010 |
| 156  | 1111111111111111 | 1111111111111111 | 0011011 | 0 | 1011001101100110 | 1100110110011011 |
| 157  | 1111111111111111 | 1111111111111111 | 0011100 | 0 | 1100001110000111 | 0000111000011100 |
| 158  | 1111111111111111 | 1111111111111111 | 0011101 | 0 | 1101001110100111 | 0100111010011101 |
| 159  | 1111111111111111 | 1111111111111111 | 0011110 | 0 | 1110001111000111 | 1000111100011110 |
| 160  | 1111111111111111 | 1111111111111111 | 0011111 | 0 | 1111001111100111 | 1100111110011111 |
| 161  | 1111111111111111 | 1111111111111111 | 0100000 | 0 | 0000010000001000 | 0001000000100000 |
| 162  | 1111111111111111 | 1111111111111111 | 0100001 | 0 | 0001010000101000 | 0101000010100001 |
| 163  | 1111111111111111 | 1111111111111111 | 0100010 | 0 | 0010010001001000 | 1001000100100010 |
| 164  | 1111111111111111 | 1111111111111111 | 0100011 | 0 | 0011010001101000 | 1101000110100011 |
| 165  | 1111111111111111 | 1111111111111111 | 0100100 | 0 | 0100010010001001 | 0001001000100100 |
| 166  | 1111111111111111 | 1111111111111111 | 0100101 | 0 | 0101010010101001 | 0101001010100101 |
| 167  | 1111111111111111 | 1111111111111111 | 0100110 | 0 | 0110010011001001 | 1001001100100110 |
| 168  | 1111111111111111 | 1111111111111111 | 0100111 | 0 | 0111010011101001 | 1101001110100111 |
| 169  | 1111111111111111 | 1111111111111111 | 0101000 | 0 | 1000010100001010 | 0001010000101000 |
| 170  | 1111111111111111 | 1111111111111111 | 0101001 | 0 | 1001010100101010 | 0101010010101001 |
| 171  | 1111111111111111 | 1111111111111111 | 0101010 | 0 | 1010010101001010 | 1001010100101010 |
| 172  | 1111111111111111 | 1111111111111111 | 0101011 | 0 | 1011010101101010 | 1101010110101011 |
| 173  | 1111111111111111 | 1111111111111111 | 0101100 | 0 | 1100010110001011 | 0001011000101100 |
| 174  | 1111111111111111 | 1111111111111111 | 0101101 | 0 | 1101010110101011 | 0101011010101101 |
| 175  | 1111111111111111 | 1111111111111111 | 0101110 | 0 | 1110010111001011 | 1001011100101110 |
| 176  | 1111111111111111 | 1111111111111111 | 0101111 | 0 | 1111010111101011 | 1101011110101111 |
| 177  | 1111111111111111 | 1111111111111111 | 0110000 | 0 | 0000011000001100 | 0001100000110000 |
| 178  | 1111111111111111 | 1111111111111111 | 0110001 | 0 | 0001011000101100 | 0101100010110001 |
| 179  | 1111111111111111 | 1111111111111111 | 0110010 | 0 | 0010011001001100 | 1001100100110010 |
| 180  | 1111111111111111 | 1111111111111111 | 0110011 | 0 | 0011011001101100 | 1101100110110011 |
| 181  | 1111111111111111 | 1111111111111111 | 0110100 | 0 | 0100011010001101 | 0001101000110100 |
| 182  | 1111111111111111 | 1111111111111111 | 0110101 | 0 | 0101011010101101 | 0101101010110101 |
| 183  | 1111111111111111 | 1111111111111111 | 0110110 | 0 | 0110011011001101 | 1001101100110110 |
| 184  | 1111111111111111 | 1111111111111111 | 0110111 | 0 | 0111011011101101 | 1101101110110111 |
| 185  | 1111111111111111 | 1111111111111111 | 0111000 | 0 | 1000011100001110 | 0001110000111000 |
| 186  | 1111111111111111 | 1111111111111111 | 0111001 | 0 | 1001011100101110 | 0101110010111001 |
| 187  | 1111111111111111 | 1111111111111111 | 0111010 | 0 | 1010011101001110 | 1001110100111010 |
| 188  | 1111111111111111 | 1111111111111111 | 0111011 | 0 | 1011011101101110 | 1101110110111011 |
| 189  | 1111111111111111 | 1111111111111111 | 0111100 | 0 | 1100011110001111 | 0001111000111100 |
| 190  | 1111111111111111 | 1111111111111111 | 0111101 | 0 | 1101011110101111 | 0101111010111101 |
| 191  | 1111111111111111 | 1111111111111111 | 0111110 | 0 | 1110011111001111 | 1001111100111110 |
| 192  | 1111111111111111 | 1111111111111111 | 0111111 | 0 | 1111011111101111 | 1101111110111111 |
| 193  | 1111111111111111 | 1111111111111111 | 1000000 | 0 | 0000100000010000 | 0010000001000000 |
| 194  | 1111111111111111 | 1111111111111111 | 1000001 | 0 | 0001100000110000 | 0110000011000001 |
| 195  | 1111111111111111 | 1111111111111111 | 1000010 | 0 | 0010100001010000 | 1010000101000010 |
| 196  | 1111111111111111 | 1111111111111111 | 1000011 | 0 | 0011100001110000 | 1110000111000011 |
| 197  | 1111111111111111 | 1111111111111111 | 1000100 | 0 | 0100100010010001 | 0010001001000100 |
| 198  | 1111111111111111 | 1111111111111111 | 1000101 | 0 | 0101100010110001 | 0110001011000101 |
| 199  | 1111111111111111 | 1111111111111111 | 1000110 | 0 | 0110100011010001 | 1010001101000110 |
| 200  | 1111111111111111 | 1111111111111111 | 1000111 | 0 | 0111100011110001 | 1110001111000111 |
| 201  | 1111111111111111 | 1111111111111111 | 1001000 | 0 | 1000100100010010 | 0010010001001000 |
| 202  | 1111111111111111 | 1111111111111111 | 1001001 | 0 | 1001100100110010 | 0110010011001001 |
| 203  | 1111111111111111 | 1111111111111111 | 1001010 | 0 | 1010100101010010 | 1010010101001010 |
| 204  | 1111111111111111 | 1111111111111111 | 1001011 | 0 | 1011100101110010 | 1110010111001011 |
| 205  | 1111111111111111 | 1111111111111111 | 1001100 | 0 | 1100100110010011 | 0010011001001100 |
| 206  | 1111111111111111 | 1111111111111111 | 1001101 | 0 | 1101100110110011 | 0110011011001101 |
| 207  | 1111111111111111 | 1111111111111111 | 1001110 | 0 | 1110100111010011 | 1010011101001110 |
| 208  | 1111111111111111 | 1111111111111111 | 1001111 | 0 | 1111100111110011 | 1110011111001111 |
| 209  | 1111111111111111 | 1111111111111111 | 1010000 | 0 | 0000101000010100 | 0010100001010000 |
| 210  | 1111111111111111 | 1111111111111111 | 1010001 | 0 | 0001101000110100 | 0110100011010001 |
| 211  | 1111111111111111 | 1111111111111111 | 1010010 | 0 | 0010101001010100 | 1010100101010010 |
| 212  | 1111111111111111 | 1111111111111111 | 1010011 | 0 | 0011101001110100 | 1110100111010011 |
| 213  | 1111111111111111 | 1111111111111111 | 1010100 | 0 | 0100101010010101 | 0010101001010100 |
| 214  | 1111111111111111 | 1111111111111111 | 1010101 | 0 | 0101101010110101 | 0110101011010101 |
| 215  | 1111111111111111 | 1111111111111111 | 1010110 | 0 | 0110101011010101 | 1010101101010110 |
| 216  | 1111111111111111 | 1111111111111111 | 1010111 | 0 | 0111101011110101 | 1110101111010111 |
| 217  | 1111111111111111 | 1111111111111111 | 1011000 | 0 | 1000101100010110 | 0010110001011000 |
| 218  | 1111111111111111 | 1111111111111111 | 1011001 | 0 | 1001101100110110 | 0110110011011001 |
| 219  | 1111111111111111 | 1111111111111111 | 1011010 | 0 | 1010101101010110 | 1010110101011010 |
| 220  | 1111111111111111 | 1111111111111111 | 1011011 | 0 | 1011101101110110 | 1110110111011011 |
| 221  | 1111111111111111 | 1111111111111111 | 1011100 | 0 | 1100101110010111 | 0010111001011100 |
| 222  | 1111111111111111 | 1111111111111111 | 1011101 | 0 | 1101101110110111 | 0110111011011101 |
| 223  | 1111111111111111 | 1111111111111111 | 1011110 | 0 | 1110101111010111 | 1010111101011110 |
| 224  | 1111111111111111 | 1111111111111111 | 1011111 | 0 | 1111101111110111 | 1110111111011111 |
| 225  | 1111111111111111 | 1111111111111111 | 1100000 | 0 | 0000110000011000 | 0011000001100000 |
| 226  | 1111111111111111 | 1111111111111111 | 1100001 | 0 | 0001110000111000 | 0111000011100001 |
| 227  | 1111111111111111 | 1111111111111111 | 1100010 | 0 | 0010110001011000 | 1011000101100010 |
| 228  | 1111111111111111 | 1111111111111111 | 1100011 | 0 | 0011110001111000 | 1111000111100011 |
| 229  | 1111111111111111 | 1111111111111111 | 1100100 | 0 | 0100110010011001 | 0011001001100100 |
| 230  | 1111111111111111 | 1111111111111111 | 1100101 | 0 | 0101110010111001 | 0111001011100101 |
| 231  | 1111111111111111 | 1111111111111111 | 1100110 | 0 | 0110110011011001 | 1011001101100110 |
| 232  | 1111111111111111 | 1111111111111111 | 1100111 | 0 | 0111110011111001 | 1111001111100111 |
| 233  | 1111111111111111 | 1111111111111111 | 1101000 | 0 | 1000110100011010 | 0011010001101000 |
| 234  | 1111111111111111 | 1111111111111111 | 1101001 | 0 | 1001110100111010 | 0111010011101001 |
| 235  | 1111111111111111 | 1111111111111111 | 1101010 | 0 | 1010110101011010 | 1011010101101010 |
| 236  | 1111111111111111 | 1111111111111111 | 1101011 | 0 | 1011110101111010 | 1111010111101011 |
| 237  | 1111111111111111 | 1111111111111111 | 1101100 | 0 | 1100110110011011 | 0011011001101100 |
| 238  | 1111111111111111 | 1111111111111111 | 1101101 | 0 | 1101110110111011 | 0111011011101101 |
| 239  | 1111111111111111 | 1111111111111111 | 1101110 | 0 | 1110110111011011 | 1011011101101110 |
| 240  | 1111111111111111 | 1111111111111111 | 1101111 | 0 | 1111110111111011 | 1111011111101111 |
| 241  | 1111111111111111 | 1111111111111111 | 1110000 | 0 | 0000111000011100 | 0011100001110000 |
| 242  | 1111111111111111 | 1111111111111111 | 1110001 | 0 | 0001111000111100 | 0111100011110001 |
| 243  | 1111111111111111 | 1111111111111111 | 1110010 | 0 | 0010111001011100 | 1011100101110010 |
| 244  | 1111111111111111 | 1111111111111111 | 1110011 | 0 | 0011111001111100 | 1111100111110011 |
| 245  | 1111111111111111 | 1111111111111111 | 1110100 | 0 | 0100111010011101 | 0011101001110100 |
| 246  | 1111111111111111 | 1111111111111111 | 1110101 | 0 | 0101111010111101 | 0111101011110101 |
| 247  | 1111111111111111 | 1111111111111111 | 1110110 | 0 | 0110111011011101 | 1011101101110110 |
| 248  | 1111111111111111 | 1111111111111111 | 1110111 | 0 | 0111111011111101 | 1111101111110111 |
| 249  | 1111111111111111 | 1111111111111111 | 1111000 | 0 | 1000111100011110 | 0011110001111000 |
| 250  | 1111111111111111 | 1111111111111111 | 1111001 | 0 | 1001111100111110 | 0111110011111001 |
| 251  | 1111111111111111 | 1111111111111111 | 1111010 | 0 | 1010111101011110 | 1011110101111010 |
| 252  | 1111111111111111 | 1111111111111111 | 1111011 | 0 | 1011111101111110 | 1111110111111011 |
| 253  | 1111111111111111 | 1111111111111111 | 1111100 | 0 | 1100111110011111 | 0011111001111100 |
| 254  | 1111111111111111 | 1111111111111111 | 1111101 | 0 | 1101111110111111 | 0111111011111101 |
| 255  | 1111111111111111 | 1111111111111111 | 1111110 | 0 | 1110111111011111 | 1011111101111110 |
| 256  | 1111111111111111 | 1111111111111111 | 1111111 | 0 | 1111111111111111 | 1111111111111111 |
