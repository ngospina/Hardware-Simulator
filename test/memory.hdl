CHIP memory {
    IN inh[16],inl[16], w, a[9];
    OUT outh[16],outl[16];

    PARTS:

        RAM512(in=inh,load=w,address=a,out=outh);
        RAM512(in=inl,load=w,address=a,out=outl);


}
