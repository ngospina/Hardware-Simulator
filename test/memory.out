| time |       inh        |       inl        |    a    | w |       outh       |       outl       |
